Dev/src/screens/TeacherScreens/ClassTabs/ClassAttendanceScreen.js

```diff
@@ -15,6 +15,7 @@ import TopBanner from 'components/TopBanner';
 import SideMenu from 'react-native-side-menu';
 import QCView from 'components/QCView';
 import screenStyle from 'config/screenStyle';
+import fontStyles from '../../../../config/fontStyles';
```

Comment:
you can directly reference 'config/fontStyles' since config is accessible directly as a module.